src/components/SRNEntry.vue

```diff
@@ -233,32 +235,21 @@ export default {
         }
       }
     },
-    /* Before an element is removed from the userIDList array, the login message and input message are reset to default values. 
-       removeInputField() is called to remove the element. 
-       In most cases, this would be enough. But there is one edge case:
-      Let's say the user typed in ID 'A' and decides to add another entry. This means 'A' is authenticated and it is valid/invalid. 
-      The user types in the ID 'B' as the next entry and it is invalid. 
-      The user changed their mind and decided to delete this entry ('B'). At this point, the variables will be in the following state:
-        validateCount = 1
-        isCurrentUserValid = false
-      The state of the variables contradict the fact that 'A' is already authenticated. This means any of the two cases:
-      - A is valid, so validateCount = 0 and isCurrentUserValid = true
-      - A is invalid, so validateCount = 2 and isCurrentUserValid = false
-      So when the user clicks submit, 'A' is authenticated again, which is a behaviour we do not want. So to bypass this, validateCount is set to 2. 
-      (To better understand how this will help, understanding processForm() will help.) 
-      isCurrentUserValid flag is ignored because the value is already set in each user's valid flag. 
-      @params {Number} - index - index of input field to be removed */
+    /** This method is called whenever - button is clicked, to remove an input field
+     * @param {Number} index - index of input field to be removed
+     */
     removeField(index) {
       this.resetInvalidInputMessage;
       this.resetInvalidLoginMessage;
       this.validateCount = 2;
       this.removeInputField(index);
     },
-    /*  this function is called whenever something is entered in the input box. It checks if the required number of characters are being typed. 
-    Until then, it prompts the user to type the required characters. 
-    If the user types more than the desired number of characters, the input is sliced and the user wont be able to see the extra characters.
-    @params {Event} - event - the event which triggered this function
-    @params {Number} - index - the index of the input field */
+
+    /** This function is called whenever something is entered in the input box.
+     * It checks if the required number of characters are being typed.
+     * @param {Event} event - the event which triggered this function
```

Comment:
```suggestion
     * @param {Object} event - the event which triggered this function
```
There's no data type as `Event` in javascript